better-link-movement-method/src/main/java/me/saket/bettermovementmethod/BetterLinkMovementMethod.java

```diff
@@ -34,17 +35,17 @@
 
     private OnLinkClickListener onLinkClickListener;
     private final RectF touchedLineBounds = new RectF();
-    private boolean isUrlHighlighted;
+    private boolean isURLHighlighted;
```

Comment:
Fixed.

I was casing acronyms like `URL` in uppercase to keep my code consistent with `URLSpan`.